components/eamxx/src/physics/p3/impl/p3_main_impl.hpp

```diff
@@ -349,7 +349,7 @@ ::p3_main(
   Int nk,
   const physics::P3_Constants<S> & p3constants)
 {
-#ifndef SCREAM_SMALL_KERNELS
+#ifndef SCREAM_P3_SMALL_KERNELS
```

Comment:
Maybe for another PR, but: how do ppl feel about swapping the code to have _positive_ CPP checks? That is
```c++
#ifdef SCREAM_P3_SMALL_KERNELS
  // do small kernels
#else
  // do monolithic
#endif
```
? My brain always has to use a few extra cycles to parse the negative, especially considering that during the first parse it never registers the `n` in the CPP pragma...

Reply:
I also prefer positive!

Reply:
can we make a rule that only positives are allowed from here on?

Reply:
I do not disagree.